runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ContactSection/ContactForm/FormFields/FormFields.module.scss

```diff
@@ -12,19 +12,19 @@ $CHECKBOX-SIZE: 26px;
 $CHECKBOX-BORDER-RADIUS: 6px;
 $CHECKBOX-TRANSFORM-Y: -1.2px;
 
-$LABEL-TRANSFORM-Y: calc($font-size-body3 / -2);
+$LABEL-TRANSFORM-Y: -25px;
```

Comment:
małe jest piękne

Reply:
seems like a bad idea, there is no other way?